### PR TITLE
[12.0][ADD] contract_fieldservice

### PR DESCRIPTION
**Pull request by murtuzasaleh**
#416 

Depends on:
- [ ] #437

### CODE REVIEW

File: oca_dependencies.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 bank-payment
 queue
-
+field-service
```
Comment (max3903) [inline oca_dependencies.txt:2]: @murtuzasaleh Can you sort the repos in alphabetic order?

File: contract_fieldservice/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Copyright 2017 Tecnativa - Luis M. Ontalba
```
Comment (max3903) [inline contract_fieldservice/__manifest__.py]:
```suggestion
# Copyright (C) 2019 Open Source Integrators
# Copyright (C) 2019 Serpent Consulting Services
```

**Review by nelsonramirezs — APPROVED** [review]

**Review by nelsonramirezs — APPROVED** [review]

**Review by max3903 — APPROVED** [review]

**Comment by max3903** [comment]
@murtuzasaleh Can you please review and merge https://github.com/murtuzasaleh/contract/pull/2?

**Review by nelsonramirezs — APPROVED** [review]
LGTM

**Review by nelsonramirezs — APPROVED** [review]

**Comment by dreispt** [comment]
Travis is red.

**Pull request closed without merging**